x-pack/plugins/siem/server/lib/detection_engine/signals/search_after_bulk_create.ts

```diff
@@ -98,13 +98,16 @@ export const searchAfterAndBulkCreate = async ({
     tags,
     throttle,
   });
-  toReturn.lastLookBackDate =
-    someResult.hits.hits.length > 0
-      ? new Date(someResult.hits.hits[someResult.hits.hits.length - 1]?._source['@timestamp'])
-      : null;
+
   if (createdItemsCount) {
     toReturn.createdSignalsCount = createdItemsCount;
   }
+  if (createdItemsCount > 0) {
```

Comment:
I think we can merge the logic in these two if statements.

Reply:
Good call - missed that. Thanks! Updated code.